login/UserResponceServices.js

```diff
@@ -0,0 +1,26 @@
+const UserResponse = require("./UserResponse.js");
+const UserModel = require("./Models/UserModel.js");
+
+module.exports.SingIn = function SingIn (loginRequest,request,response)
+{
+    UserModel.User.findOne({raw:true,where: {login: loginRequest.login,password:loginRequest.password}})
+        .then(User=>{
+            if(!User){
+                console.log("Данные введены неправильно");
+                return;
+            }
+            else {
+                let userTables = JSON.parse(JSON.stringify(User));
```

Comment:
if you don't modify variable pls use const instead let

Reply:
yes,that was mistake